.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "no-inline-html": false,
+  "no-bare-urls": false,
+  "line-length": false,
+  "first-line-h1": false,
+  "single-h1": false,
+  "no-duplicate-header": false,
+  "no-alt-text": false,
+  "no-emphasis-as-header": false,
+  "ol-prefix": false,
+  "no-trailing-punctuation": false
+}
```

Comment:
How about adding:

```txt
  "list-marker-space": { "ol_multi": 2, "ol_single": 2 },
  "ol-prefix": { "style": "one" }
```
?

I was thinking of suggesting them later, but because we have changes in a lot of lists anyway, now can be a good opportunity. If adding the rules does not make `git diff` much worse, we can save ourselves from ‘breaking’ `git blame` again in the future.

Reply:
I tried this, but adding `"ol-prefix": { "style": "one" }` will require manual edits in a lot of files, so I recommend fixing it later as we edit those files.

Reply:
Also I recommend local testing after making this change, since I'm not sure yet how our specific mdx-parser will handle this change so capturing it as a separate task: https://app.asana.com/0/1199550852792212/1201333602499681/f